frontend/encore/babel.rst

```diff
@@ -7,6 +7,10 @@ Configuring Babel
 
 Need to extend the Babel configuration further? The easiest way is via
 ``configureBabel()``:
+But first let's install the babel preset.
+
+``yarn add babel-preset-es2017``
+This will install the preset that we will use later to configure the babel.
```

Comment:
I think this is indeed a good clarification! But what about adding this information as comments in the code block below (right above the es2017 line. We can say something like:

// make sure to install any presets you want to use
// yarn add ...